kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -11,36 +11,71 @@
     text-overflow: ellipsis;
 }
 
+.kb-data-breadcrumb a {
+    cursor: pointer;
+}
+
 .kb-data-staging-footer {
-    font-family : Oxygen, Arial, sans-serif;
+    font-family: Oxygen, Arial, sans-serif;
     font-weight: bold;
-    font-size : 105%;
-    text-align : center;
-    background-color : #EEEEEE;
-    height : 32px;
-    padding : 8px;
+    font-size: 105%;
+    text-align: center;
+    background-color: #EEEEEE;
+    height: 32px;
+    padding: 8px;
 }
 
 .kb-data-staging-metadata-list {
-  font-family : Oxygen;
-  font-weight: bold;
-  width : 115px;
-  display : inline-block;
-  font-size : 90%;
+    font-family: Oxygen, Arial, sans-serif;
+    font-weight: bold;
+    width: 115px;
+    display: inline-block;
+    font-size: 90%;
 }
 
 .kb-data-staging-metadata-file-lines {
-  white-space : pre;
- overflow : scroll;
- font-family : monospace;
- font-size : 90%;
- background-color : #DDDDDD;
- max-height : 200px;
+    white-space: pre;
+    overflow: scroll;
+    font-family: monospace;
+    font-size: 90%;
+    background-color: #DDDDDD;
+    max-height: 200px;
+}
+
+/* https://www.figma.com/file/T30UN3QhWhhV5SpXiqY1ij/KBase-component-library?node-id=1%3A4 */
+.kb-data-staging-button {
+    background: none;
+    border: none;
+    cursor: pointer;
+    font-family: Oxygen, Arial, sans-serif;
```

Comment:
The typography in [figma](https://www.figma.com/file/zBQcHiLV4FlIkD3Y1ogNcy/concept-sketches2) only has oxygen.